azerothcore/azerothcore-wotlk
# feat(Core/Conf): Rate.RewMoneyMaxLevel

## Changes Proposed:
-  Make the EXTRA amount of money you get from quests when you are at the max level configurable
-  Code cleanup (remove code repetition, use static casts, etc..)


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4497



## Tests Performed:
- Tested in game



## How to Test the Changes:

- Create a char and bring it to max level (by default, max level is 80)
- Try to complete quest that give an extra reward at max level
- Try to change the rates and see how the rewarded money amount changes



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
